Test/Machine/test_machine.py

```diff
@@ -1,96 +1,111 @@
 import netket as nk
 import networkx as nx
 import numpy as np
+import pytest
 from mpi4py import MPI
 
 machines = {}
 
+
 # TESTS FOR SPIN HILBERT
 # Constructing a 1d lattice
-g = nk.graph.Hypercube(length=20, ndim=1)
+g = nk.graph.Hypercube(length=4, ndim=1)
 
 # Hilbert space of spins from given graph
-hi1 = nk.hilbert.Spin(s=0.5, graph=g)
-
-machines["RbmSpin 1d Hypercube spin"] = nk.machine.RbmSpin(hilbert=hi1, alpha=1)
+hi = nk.hilbert.Spin(s=0.5, graph=g)
 
-machines["RbmSpinSymm 1d Hypercube spin"] = nk.machine.RbmSpinSymm(
-    hilbert=hi1, alpha=2)
+machines["RbmSpin 1d Hypercube spin"] = [nk.machine.RbmSpin(
```

Comment:
Here I pack together machines and respective hilberts. Maybe this can be done in a nicer way in python

Reply:
I think this is a fine way to do it.